text/0060-tasks-api.md

```diff
@@ -315,9 +375,9 @@ The auth layer can return the following errors if Meilisearch is secured (a mast
 
 ---
 
-**Get a task by uid** | `GET` - `/tasks/{uid}`
+##### 6.2. Get a task by uid | `GET` - `/tasks/{uid}`
```

Comment:
We should mention that there are query params and that they are the same as `GET /tasks`